packages/sanity/src/core/form/inputs/PortableText/object/AnnotationToolbarPopover.tsx

```diff
@@ -209,6 +209,7 @@ export function AnnotationToolbarPopover(props: AnnotationToolbarPopoverProps) {
       referenceBoundary={referenceBoundary}
       referenceElement={cursorElement}
       scheme={popoverScheme}
+      {...(annotationOpen ? {width: 1} : {})}
```

Comment:
We don't want the popover shown before you open the annotation (to edit and delete it) to be wider